build.sbt

```diff
@@ -26,7 +31,9 @@ def commonSettings: Seq[Setting[_]] = Seq(
   testOptions += Tests.Argument(TestFrameworks.ScalaCheck, "-w", "1"),
   javacOptions in compile ++= Seq("-target", "6", "-source", "6", "-Xlint", "-Xlint:-serial"),
   incOptions := incOptions.value.withNameHashing(true),
-  crossScalaVersions := Seq(scala210)
+  crossScalaVersions := Seq(scala210),
+  bintrayPackage := (bintrayPackage in ThisBuild).value,
+  bintrayRepository := (bintrayRepository in ThisBuild).value
```

Comment:
I'm still not getting it :-/

Is this overriding some other setting?
Or is it a stub for this changing in the near future?

It just looks like a noop to me..